.github/workflows/build.yml

```diff
@@ -32,6 +32,9 @@ jobs:
       - name: Install Python dependencies
         run: |
           pip install ".[test]"
+      - name: Build Jupyter Enterprise Gateway conda env
+        run: |
+          SA="source $CONDA_HOME/bin/activate" make env
```

Comment:
I don't think we need to build an env as part of the CI since none of the targets use it explicitly.